s2core/src/test/scala/org/apache/s2graph/core/CommonTest.scala

```diff
@@ -0,0 +1,13 @@
+package org.apache.s2graph.core
+
+import org.scalatest.Tag
+
+/**
```

Comment:
guess we need to remove auto generated author field. I know other sources have it now, but they also need to be removed eventually.